fixpatches/build.gradle.kts

```diff
@@ -5,13 +5,26 @@ plugins {
     `maven-publish`
 }
 
+repositories {
+    mavenCentral()
+    maven {
+        name = "saveourtool/okio-extras"
+        url = uri("https://maven.pkg.github.com/saveourtool/okio-extras")
+        credentials {
+            username = project.findProperty("gprUser") as String? ?: System.getenv("GITHUB_ACTOR")
```

Comment:
now we need to provide `gprUser` and `gprKey` in our local configuration, to make it work, right?
could we use simple maven repo instead of github packages?

Reply:
We need indeed.

Please up-vote saveourtool/okio-extras#20.